Gemfile.lock

```diff
@@ -292,20 +292,20 @@ GEM
       rspec-mocks (~> 3.9)
       rspec-support (~> 3.9)
     rspec-support (3.9.3)
-    rubocop (0.90.0)
+    rubocop (0.93.1)
       parallel (~> 1.10)
-      parser (>= 2.7.1.1)
+      parser (>= 2.7.1.5)
       rainbow (>= 2.2.2, < 4.0)
-      regexp_parser (>= 1.7)
+      regexp_parser (>= 1.8)
       rexml
-      rubocop-ast (>= 0.3.0, < 1.0)
+      rubocop-ast (>= 0.6.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 2.0)
-    rubocop-ast (0.4.0)
-      parser (>= 2.7.1.4)
+    rubocop-ast (1.0.1)
+      parser (>= 2.7.1.5)
     rubocop-performance (1.8.0)
       rubocop (>= 0.87.0)
-    rubocop-rails (2.8.0)
+    rubocop-rails (2.8.1)
```

Comment:
Needed to update rubocop-rails since it had a bug which has been fixed in the recent patch release